pallets/eth-bridge/src/lib.rs

```diff
@@ -556,6 +549,69 @@ pub mod pallet {
 
             Ok(().into())
         }
+
+        #[pallet::call_index(7)]
+        #[pallet::weight(<T as Config>::WeightInfo::add_eth_tx_hash())]
```

Comment:
```suggestion
         // TODO: Benchmark me
        #[pallet::weight(0)]
```

Reply:
it was fixed on the next PR but will backport here